### PR TITLE
Mention how to use Docker for building documentation

### PR DESCRIPTION
**Pull request by bquorning**
I didn’t want to install Antora and npm on my computer, so I tried using Docker instead.

### CODE REVIEW

**Comment by koic** [comment]
I have not deployed the docs with Docker, but this is one of methods. So if you can confirm the operation, there is no problem with this :-)

File: README.adoc [hunk 1]
```diff
@@ -65,6 +65,18 @@ NOTE: You'll need commit access to the repository for this to work.
...
+
+If you prefer not to install Antora on your local machine, you can build the documentation
+inside a Docker container like this:
+
+[source]
+----
+$ docker run -t -v $(pwd):/docs antora/antora:3.1.2 --fetch /docs/antora-playbook.yml
+----
+
+Then, procedd with `git add`, `git commit` and `git push` as mentioned above.
```
Comment (koic) [inline README.adoc]: Is this a typo?

```suggestion
Then, proceed with `git add`, `git commit` and `git push` as mentioned above.
```

**Comment by bquorning** [comment]
> So if you can confirm the operation, there is no problem with this :-)

That is how I created the latest documentation that I just pushed. I hope it works 😅

File: README.adoc (hunk 1 above)
Comment (bquorning) [reply README.adoc]: Yes, typo. Thank you for catching it.

**Review by koic — APPROVED** [review]
Looks good to me! 👍

**Review by bbatsov — APPROVED** [review]

**Pull request merged**